containers/base/Dockerfile

```diff
@@ -130,15 +130,16 @@ RUN pip install google-cloud-dataflow==0.4.1
 ADD nbconvert /datalab/nbconvert
 ADD config/ipython.py /etc/ipython/ipython_config.py
 ADD config/nbconvert.py /etc/jupyter/jupyter_notebook_config.py
+# Directory "py" may be empty and in that case it will git clone pydatalab from repo
+ADD py /datalab/lib/pydatalab
```

Comment:
I wish that could work so we can simplify the build.sh. It seems "ADD py*" will fail the docker build if py* does not exist. Neither does "COPY py*" work:
https://github.com/docker/docker/issues/13045
So we may have to create an empty py dir. I'll keep looking for a better approach but if you have any ideas please let me know!